File: PV239/seminar2/src/main/java/cz/pv239/seminar2/FragmentActivity.java
package cz.pv239.seminar2;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.FragmentManager;
import android.support.v7.app.AppCompatActivity;

public class FragmentActivity
        extends AppCompatActivity {

    private static final String TAG = FragmentActivity.class.getSimpleName();

    @NonNull
    public static Intent newIntent(@NonNull Context context) {
        return new Intent(context, FragmentActivity.class);
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_fragment);

        FragmentManager fragmentManager = getSupportFragmentManager();
        if (fragmentManager == null) {
            return;
        }

        fragmentManager.beginTransaction().add(R.id.content1,
                FirstFragment.newInstance(),
                FirstFragment.class.getSimpleName()).commit();
        fragmentManager.beginTransaction().add(R.id.content2,
                SecondFragment.newInstance(),
                SecondFragment.class.getSimpleName()).commit();
    }
}

File: PV239/seminar2/src/main/java/cz/pv239/seminar2/FragmentActivity.kt
package cz.pv239.seminar2

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class FragmentActivity : AppCompatActivity() {

    companion object {

        fun newIntent(context: Context): Intent {
            return Intent(context, FragmentActivity::class.java)
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_fragment)

        val fragmentManager = supportFragmentManager ?: return

        // Put a new instance of FirstFragment to FrameLayout with id R.id.content1
        fragmentManager.beginTransaction().add(R.id.content1,
                FirstFragment.newInstance(),
                FirstFragment::class.java.simpleName).commit()

        fragmentManager.beginTransaction().add(R.id.content2,
                SecondFragment.newInstance(),
                SecondFragment::class.java.simpleName).commit()
    }
}
